### DISCUSSION TITLE
Matching step (HC) and Double Metaphone URI

### DISCUSSION THREAD

**Discussion by radaniba** [marklogic-community/envision#255] [Q&A]
Hi there,

I am trying to configure a Matching step on firstName and lastName attributes using Double Metaphone as Match Type but in HC it is asking the dictionnary URI which I am not sure how to refer to by default in HC, should I download it and install it ? the documentation is a bit confusing about it unfortunately, it explains 'what it is' but doesn't really guide through how to install/configure it (if we have to)

Testing the javascript in the console works though, so I am assuming there is a dictionary loaded by default, but can't get through the URI  :/

**Comment by wpaven** [comment]
I'd start with the HC documentation.  CE is basically providing an alternative view over the same implementation so you don't have to bounce between apps so we rely on the HC docs here.   https://docs.marklogic.com/datahub/5.4/flows/about-mastering.html

You have to create your own dictionary. Nothing is installed by default. 

But even the HC documentation is lacking.  So I'd suggest posting this question on stack overflow as I'm not sure what the latest guidance is myself.
